src/components/menu-list/menu-list-renderer.tsx

```diff
@@ -70,8 +55,8 @@ export class MenuListRenderer {
         return (
             <ul
                 class={classNames}
-                aria-hidden={(this.config.type === 'menu').toString()}
-                role={role}
+                aria-hidden={true}
+                role="menu"
```

Comment:
Some of the changes to this file belongs to the third commit right? Since they are menu related.

Reply:
Hmm… yeah, that's weird… I must have mixed something up while juggling different fixups or something… Thanks for the catch! 🙏

Reply:
⚡